linchpin/provision/roles/azure/tasks/provision_azure_api.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Use Azure Api to manage resources
+  azure_rm_resource:
+    client_id: "{{ auth_var['client_id'] | default(omit) }}"
+    tenant: "{{ auth_var['tenant'] | default(omit) }}"
+    secret: "{{ auth_var['secret'] | default(omit) }}"
+    subscription_id: "{{ auth_var['subscription_id'] | default(omit) }}"
+    resource_group: "{{res_def['resource_group']|default(omit)}}"
+    provider: "compute"
```

Comment:
azure_rm_resource allows any resource to be created, why is it limited to "compute"?